### ISSUE TITLE
Viewer not loading

### ISSUE THREAD

**Issue by tuxedobyrd** [Daniel-KM/UniversalViewer4Omeka#2]
I have followed all the steps, and all I get on my item/show page is a big, black box. Any suggestions?

**Comment by Daniel-KM** [comment]
Hi,

If you see a black box, that means that the viewer is well installed. So this is probably a issue related to javascript, either with another plugin, or with the theme. But hard to say more without a message. Is there a message in the javascript console (ctrl+shift+K on Firefox)? 

Sincerely,

Daniel Berthereau
Infodoc & Knowledge management

**Comment by Daniel-KM** [comment]
Hi,

If not related to javascript, it can be: no image processor (gd or imagick), unsupported format (tiff is not supported by gd, etc.).

Anyway, I just published an update of the plugin with the library already included.

Sincerely,

Daniel Berthereau
Infodoc & Knowledge management

**Comment by tuxedobyrd** [comment]
Thank you for your responses, Daniel. I have installed the new version, but it did not fix the issue. This is the message in the javascript console:

http://greenwoodhistorical.org/catalog/plugins/UniversalViewer/views/shared/javascripts/uv/lib/embed.js Failed to load resource: the server responded with a status of 404 (Not Found).

I also tried deactivating every other plugin to see if it was a plugin conflict, but that did not help.

I have a different, local, installation of Omeka on my Mac, using MAMP, and the viewer loads fine in that. But it doesn't load on either of the two Omeka sites I have hosted on shared servers. Could it be related to the server?
